index.html

```diff
@@ -1,104 +1,135 @@
-                <div class="flex-item2"><h2>Multi-Post Stories Gain+Glory</h2></div>
-                <ul class="inner-flex-item">
+                    <button class="flex-item4">See Project</button>
+                </div>
+            </article>
+            <article class="main-container">
+                <div class="grid-item">
+                    <div class="flex-item1"><img src="img/Plaholder.png" alt="placeholder"></div>
+                    <div class="flex-item2">
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                    </div>
+                    <ul class="inner-flex-item">
                         <li class="pill-btn">Ruby on rails</li>
                         <li class="pill-btn">css</li>
                         <li class="pill-btn">javaScript</li>
                         <li class="pill-btn">html</li>
                     </ul>
-                <button class="flex-item4">See Project</button>
-            </div>
-        </article>
-        
-    </div> 
+                    <button class="flex-item4">See Project</button>
+                </div>
+            </article>
+
+        </div>
     </section>
-    
-        <section class="aboutme-container">
-            <div class="wrap">
-                <h2 class=" box-item headline"> About Me</h2>
-                <p class="box-item">  I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don’t hestiate to contact me.</p> 
-                <button class="fixx">Get My Resume</button>
-            </div>       
-        </section>
-    
+
+    <section class="form">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form class="form-container" action="https://formspree.io/f/xjvlbpry" method="POST">
+
+            <input required class="form-item" maxlength="30" type="text" id="fullName" name="full_name"
+                placeholder="Full Name">
+
+            <input required class="form-item" type="email" id="mail" name="user_email" placeholder="Email Address">
+
+            <textarea required class="form-item" maxlength="500" id="msg" name="message_area"
+                placeholder="Enter test here"></textarea>
+
+            <button class="flex-item4" type="submit">Get in touch</button>
+
+
+        </form>
+        <ul class="social-icons">
+            <li> <a href="#"> <img src="img/twitter.svg" alt="twitter"></a></li>
+            <li> <a href="#"> <img src="img//Linkedin-icon.svg" alt="linkedin"></a></li>
+            <li> <a href="#"> <img src="img/medium.png" alt="medium"></a></li>
+            <li> <a href="#"> <img src="img/github.svg" alt="github"></a></li>
+            <li> <a href="#"> <img src="img/angellist.png" alt="angellist"></a></li>
+        </ul>
+        <p>a.alrakhawy@gmail.com</p>
```

Comment:
- [ ] [OPTIONAL]: To make your project even more professional, you should consider making this an `<a>` with the href attribute set your own email, this way, whenever the user clicks on this link, it will be redirected to a black email, with your email as recipient. You can read more about it [here](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#e-mail_links).

Reply:
I will do it, thanks 😃